templates/zerver/help/preview-your-message-before-sending.md

```diff
@@ -1,11 +1,15 @@
 # Previewing your message
 
-You can preview your message before you send it by clicking the **eye** (![eye](/static/images/help/eye.png)) icon.
+You can preview your message before you send it by clicking the **eye**
+(![eye](/static/images/help/eye.png)) icon at the bottom of the **Compose message box**.
 
 ![Original content](/static/images/help/before-preview.png)
 
+This is bring up the message preview:
+
 ![Previewed](/static/images/help/during-preview.png)
 
-To resume editing, click the **pencil and paper** (![pencil and paper](/static/images/help/pencil-paper.png)) icon that has replaced the eye icon.
+To resume editing, click the **pencil and paper** (![pencil and paper](/static/images/help/pencil-paper.png))
```

Comment:
Don't think this should be a step. Isn't really a step into previewing a message, but rather a return to  original.